assets/css/styleguide.scss

```diff
@@ -77,9 +77,9 @@ $site-top:           124px;
       float: right;
       display: none;
       margin-top: -5px;
-      
-      li { 
-        display: inline; 
+
+      li {
```

Comment:
I can't find the matching brace for this. It's either missing the closing brace or the alignment is off.